config/java.util.logging.properties

```diff
@@ -54,6 +54,9 @@ handlers=java.util.logging.ConsoleHandler, org.apache.ignite.logger.java.JavaLog
 #
 #org.apache.ignite.CourtesyConfigNotice.level=OFF
 
+org.springframework.level=WARNING
```

Comment:
Why are these properties at the WARNING level? I think it would be better to use ERROR. This makes it easier to debug problems.

Reply:
This was copied from `config/ignite-log4j2.xml`, `config/ignite-log4j.xml` Should we change this too?

Reply:
ERROR will show fewer issues than WARN. Will it really make debugging easier?

Reply:
I guess that ERROR shows warns but WARN doesn't show errors. Please correct me if I am wrong.

Reply:
No, this level work as threshold. So WARN will be include ERROR and WARN, ERROR will include only ERROR